### DISCUSSION TITLE
Inquiry on the Possibility of Using .wasm Files in iOS Apps

### DISCUSSION THREAD

**Discussion by qorzi** [nodejs-mobile/nodejs-mobile#91] [Q&A] [answered]
Hello,

I am currently facing difficulties in using .wasm files within an iOS app. It seems that due to the --jitless mode in iOS, the use of the WebAssembly object is not possible. I am curious if this makes the execution of .wasm files entirely impossible within the app, and if there is any way to utilize them despite these limitations.

If anyone has experienced similar challenges or knows of a solution to this issue, I would greatly appreciate your advice. Sharing information on the restrictions of using .wasm files in iOS and any potential workarounds would be incredibly helpful.

Thank you.

**Comment by staltz** [comment, marked as answer]
I can confirm it's not possible, it's something that would have been quite beneficial for some projects where I use nodejs-mobile.

> It seems that due to the --jitless mode in iOS, the use of the WebAssembly object is not possible.

And yes, this is the reason why.
